olaf/_internals/services/logs.py

```diff
@@ -8,14 +9,21 @@
 from ...common.oresat_file import new_oresat_file
 from ...common.service import Service
 
-TMP_LOGS_FILE = "/tmp/olaf.log"
+_logs = []
 
 
-def logger_tmp_file_setup(level: str):
-    """Get the boot log file handler and clean up temp log file used by LogsService."""
-    if os.path.isfile(TMP_LOGS_FILE):
-        os.remove(TMP_LOGS_FILE)
-    logger.add(TMP_LOGS_FILE, level=level, backtrace=True)
+class LogServiceHandler(logging.Handler):
+    """Custom Log handler that doesn't log to file."""
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.data = []
```

Comment:
I assume `self.data` was left over from trying to make this work without a global  `_logs`? If `on_read_since_boot` is moved to this class then this can stay, otherwise the whole overridden `__init__()` can be removed. (I'd obviously prefer the former.)